.claude/templates/decision-log.md

```
@@ -0,0 +1,199 @@
+| Total Decisions | `___` |
+| ACCEPTED | `___` |
+| REJECTED | `___` |
+| DEFERRED | `___` |
+| SUPERSEDED | `___` |
+
+---
+
+## Decision Categories
+
+### Architecture Decisions
+| ID | Title | Status | Date |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)